app/src/main/res/layout/activity_main.xml

```diff
@@ -8,12 +8,7 @@
 
     <include
```

Comment:
<b>Reporter: </b>Android Lint<br /><b>Violation: </b>MissingConstraints<br /><b>Notes: </b>This view is not constrained. It only has designtime positions, so it will jump to (0,0) at runtime unless you add the constraints